### PR TITLE
View & add multiple images to an event

### PR DESCRIPTION
**Pull request by sukhada**
@caro5 

screen shot 2016-09-24 at 1 50 15 pm

### CODE REVIEW

File: app/src/main/java/turtler/voyageur/adapters/EventAdapter.java [hunk 1]
```diff
@@ -87,7 +92,7 @@ public void onClick(View view) {
                 }
             } catch (ParseException e) {
                 e.printStackTrace();
-            }
+            }*/
```
Comment (caro5) [inline app/src/main/java/turtler/voyageur/adapters/EventAdapter.java:95]: ?

File: app/src/main/java/turtler/voyageur/adapters/EventAdapter.java [hunk 2]
```diff
@@ -72,6 +73,10 @@ public void onClick(View view) {
 
             FragmentActivity activity = (FragmentActivity)(mContext);
             FragmentManager fm = activity.getSupportFragmentManager();
+            DetailEvent alertDialog = DetailEvent.newInstance(event.getObjectId());
+            alertDialog.show(fm, "fragment_detail_event");
+
+            /*
```
Comment (caro5) [inline app/src/main/java/turtler/voyageur/adapters/EventAdapter.java:79]: remove

File: app/src/main/java/turtler/voyageur/fragments/DetailEvent.java [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+            }
+        }
+        sliderLayout.setVisibility(View.VISIBLE);
+
+        tvCaption.setText(ev.getCaption());
+        if (ev.getDate() != null) {
+            tvDate.setText(TimeFormatUtils.dateTimeToString(ev.getDate()));
+        }
+        /*
+        tvTitle.setOnClickListener(new View.OnClickListener() {
```
Comment (caro5) [inline app/src/main/java/turtler/voyageur/fragments/DetailEvent.java:159]: we can just delete this

Comment (sukhada) [reply app/src/main/java/turtler/voyageur/fragments/DetailEvent.java:159]: we still need some way to connect to the map marker though...was planning on adding that back in some form in a future PR

File: app/src/main/java/turtler/voyageur/adapters/EventAdapter.java (hunk 1 above)
Comment (sukhada) [reply app/src/main/java/turtler/voyageur/adapters/EventAdapter.java:95]: we still need some way to connect to the map marker though...was planning on adding that back in some form in a future PR

**Comment by caro5** [comment]
lgtm

**Pull request merged**